test_bandit.py

```
@@ -0,0 +1,189 @@
+    config = {"password": "secret123"}  # S108
+    
+    # S110: Try, except, pass detected
+    try:
+        pass
+    except:
+        pass  # S110
+    
+    # S112: Try, except, continue detected
+    try:
+        pass
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)